src/Storage/Authorization/AuthorizationService.cs

```diff
@@ -152,6 +149,48 @@ public async Task<bool> AuthorizeInstanceAction(Instance instance, string action
             return authorized;
         }
 
+        /// <inheritdoc/>>
+        public async Task<bool> AuthorizeAnyOfInstanceActions(Instance instance, List<string> actions, string task = null)
```

Comment:
You're trying to make the method I decided not to make when I worked on the AllowNewCopy change. See PR #176 .

I would sleep better if we used the `CreateMultiDecisionRequest` method to create the request.

Reply:
I had a look at that method, but as this is not making a single decision and not multiple I tested this solution, but might be the auth mock that is behaving different than the real auth